sdk/storage/azure-storage-blob-nio/README.md

```diff
@@ -19,7 +19,7 @@ This package allows you to interact with Azure Blob Storage through the standard
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-storage-blob-nio</artifactId>
-    <version>12.0.0-beta.7</version>
+    <version>12.0.0-beta.9</version>
```

Comment:
similar comments like for changefeed.

Reply:
confirmed [beta8](https://search.maven.org/artifact/com.azure/azure-storage-blob-nio/12.0.0-beta.8/jar) was also released here. I think this was also a readme oversight.